### PR TITLE
✨ Adding "Browsi" amp-analytics & RTC vendors

### PR DESCRIPTION
**Pull request by ofirpaBrowsi**
The analytics and RTC vendors additions provide the option to send real time data to ad objects on AMP pages using Browsi Prediction Service.

### CODE REVIEW

**Comment by ofirpaBrowsi** [comment]
@lannka Can you please review this?

**Comment by jeffjose** [comment]
Over to @zhouyx for review.

**Comment by zhouyx** [comment]
I know @lannka has reviewed the I2I and design. I'll have him review the PR, but I'm happy to help review the analytics config part. 

@ofirpaBrowsi before I proceed, could you please sync to latest master and resolve the merge conflict. We recent launched a code improvement and moved all vendor configs from the .js file to a .json file, so that those configs can be lazily loaded. It's more scalable this way and also reduces the component file size.

**Comment by ofirpaBrowsi** [comment]
@zhouyx  Done.

File: extensions/amp-analytics/0.1/vendors/browsi.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+/**
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors/browsi.js]: Please remove the file. Only browsi.json is needed. Thanks.

File: extensions/amp-analytics/0.1/vendors/browsi.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      }
+    },
+    "v100-0": {
+      "on": "visible",
+      "request": "visibility",
+      "selector": "amp-ad[data-amp-slot-index='0']",
+      "visibilitySpec": {
+        "visiblePercentageMin": 100,
+        "continuousTimeMin": 1000,
+        "repeat": false
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors/browsi.json]: nit: You don't need this `repeat` is `false` by default.

File: extensions/amp-analytics/0.1/vendors/browsi.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      },
+      "request": "engagement",
+      "vars": {
+        "type": "engagement"
+      }
+    },
+    "v100-0": {
+      "on": "visible",
+      "request": "visibility",
+      "selector": "amp-ad[data-amp-slot-index='0']",
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors/browsi.json]: Want to confirm this is the desired behavior, to always select the first ad slot?

File: examples/analytics-vendors.amp.html [hunk 4]
```diff
@@ -335,6 +336,18 @@
 </amp-analytics>
 <!-- Baidu stats example -->
 
+  <!-- Browsi prediction tracking example -->
+  <amp-analytics type="browsi" id="browsi">
```
Comment (zhouyx) [inline examples/analytics-vendors.amp.html]: FYI: The example is throwing an error that " Element "amp-ad[data-amp-slot-index='0']" not found: null​​​". You can fix it by overriding the default config. I'm fine with leaving the error as it is.

**Comment by zhouyx** [comment]
@ofirpaBrowsi Looks great! PR merged! Thanks

**Pull request merged**